pulsar-broker/src/test/java/org/apache/pulsar/client/api/ConsumerRedeliveryTest.java

```diff
@@ -53,7 +53,7 @@ public class ConsumerRedeliveryTest extends ProducerConsumerBase {
     @BeforeClass
     @Override
     protected void setup() throws Exception {
-        conf.setManagedLedgerCacheEvictionFrequency(0.1);
+        conf.setManagedLedgerCacheEvictionIntervalMs(10000);
```

Comment:
We are compatible with this configuration. Why need to change this line?

Reply:
Yes, these two configurations are compatible. 
I think it's better to cover the newly added configurations than the deprecated one.

Reply:
Got it, thanks!